docs/en/engines/table-engines/integrations/azureBlobStorage.md

```diff
@@ -30,6 +30,7 @@ CREATE TABLE azure_blob_storage_table (name String, value UInt32)
 - `account_key` - if storage_account_url is used, then account key can be specified here
 - `format` — The [format](/interfaces/formats.md) of the file.
 - `compression` — Supported values: `none`, `gzip/gz`, `brotli/br`, `xz/LZMA`, `zstd/zst`. By default, it will autodetect compression by file extension. (same as setting to `auto`).
+- `extra_credentials` - Use `client_id` and `tenant_id` for authentication instead of `account_name` and `account_key`.                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
Worth adding that 
"If extra_credentials are provided, they are given priority over account name & account key"